pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBase.java

```diff
@@ -884,7 +884,7 @@ protected void callMessageListener(Message<T> msg) {
         }
     }
 
-    protected static final byte[] NONE_KEY = "NONE_KEY".getBytes(StandardCharsets.UTF_8);
+    private static final byte[] NONE_KEY = "NONE_KEY".getBytes(StandardCharsets.UTF_8);
```

Comment:
I don't know why, but it seems that compiling will fail if this field is protected.

> Error:  Medium: org.apache.pulsar.client.impl.ConsumerBase.NONE_KEY should be package protected [org.apache.pulsar.client.impl.ConsumerBase] At ConsumerBase.java:[line 887] MS_PKGPROTECT